### PR TITLE
Update README with Cloud Run integration

### PR DESCRIPTION
**Pull request by sanguinecloud**
Editing the VS Code marketplace description to include Cloud Run.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,30 @@
...
-like local development, shared development, test, or production, so you can
-easily test and debug on your workstation or in the cloud.
+Cloud Code for VS Code brings the power and convenience
+of IDEs to developing cloud-native Kubernetes and Cloud Run applications.
+Cloud Code works with Google’s command-line [container tools][1] like [`skaffold`][2]
+and [`kubectl`][3] under the hood, providing local, continuous feedback on your project
+as you build, edit, and run your applications on your workstation or in the cloud.
+Additionally, its support for [deployment profiles][4] lets you define different
+environments, like local development, shared development, test, and production, to
+easily test and debug your app with, locally or in the cloud.
```
Comment (sujit-kamireddy) [inline README.md:10]: I think we can remove the deployment profiles part. We moved away from those notion.

File: README.md [hunk 2]
```diff
@@ -1,31 +1,30 @@
...
 
 ## Key Features
 
 - Support for Go, Node, Java, and Python
-- Rapid Edit, Package, Deploy loop to your K8s cluster
 - Integrated Debugging and Log Viewing/Streaming
-- Snippets, completions, and linting for K8s artifacts
-- Profile support for dev, test and production environments
-- Cluster management, resource browsing and inspection of K8s clusters
+- Profile support for dev, test, and production environments
```
Comment (sujit-kamireddy) [inline README.md:23]: I think we should remove this.

File: README.md (hunk 1 above)
Comment (sanguinecloud) [reply README.md]: ah, yes, done!

File: README.md (hunk 2 above)
Comment (sanguinecloud) [reply README.md]: done!

**Pull request merged**